.env

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY = "TOKENHELLOWORLD!!"
```

Comment:
You should not commit the .env file to github. Add it to the gitignore

Reply:
Yes, I am going to do it. I am going to do it, I thought I did.